src/script/ceph-backport.sh

```diff
@@ -37,7 +37,11 @@
 # redmine_user_id=[your_redmine_user_id]
 # github_token=[your_github_personal_access_token]
 # github_user=[your_github_username]
-# 
+#
+# you can also optionally add the remote repo's name in this file, like
+#
+# github_repo=[your_github_repo_name]
```

Comment:
Please add: If you don't add it, it will default to "origin"

Reply:
updated and repushed.